### PR TITLE
Pulled some stuff from Reddwarf

### PR DESCRIPTION
**Pull request by TimSimpsonR**
- Added a script to create a VM.
- Added a script to control a dev environment (similar to reddwarf-ci).
- Added the example generator.

### CODE REVIEW

**Comment by hub-cap** [comment]
wow cool thats a ton of stuff!!! lets discuss tomorrow am!!

File: integration/reddwarf-dm [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+    echo "This script is meant to run in a disposable dev environment. "
+    echo "Run create_dm_vm to create such a machine."
+    exit 1
+fi
+
+# Load global configuration variables.
+source /integration/reddwarf-dm.rc
+
+
+NOVA_HOME="/src"
```
Comment (hub-cap) [inline integration/reddwarf-dm]: remove me plz

File: integration/conf.json.example [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "devstack":"/home/yourname/work/devstack",
```
Comment (hub-cap) [inline integration/conf.json.example]: Id null all these out by default, so we can make the "happy path" be without any code.

Comment (TimSimpsonR) [inline integration/conf.json.example]: Fixed.

File: integration/reddwarf-dm (hunk 1 above)
Comment (TimSimpsonR) [inline integration/reddwarf-dm]: Fixed.

**Pull request merged**